pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["Cython~=3.0.11", "setuptools>=64", "setuptools-scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fugashi"
+description = "Cython MeCab wrapper for fast, pythonic Japanese tokenization."
+readme = "README.md"
+requires-python = ">=3.8"
+license.text = "MIT"
```

Comment:
This setting is legacy.
However, the new syntax requires setuptools>=77 and python>=3.9.

If you drop python3.8 you should rewrite it as follows:
```diff
[build-system]
-requires = ["Cython~=3.0.11", "setuptools>=64", "setuptools-scm>=8"]
+requires = ["Cython~=3.0.11", "setuptools>=77", "setuptools-scm>=8"] 
```

```diff
-requires-python = ">=3.8"
-license.text = "MIT"
+requires-python = ">=3.9"
+license = "MIT AND BSD-3-Clause"
+license-files = ["LICENSE", "LICENSE.mecab"]
```

```diff
-  "License :: OSI Approved :: MIT License",
```

```diff
[tool.setuptools]
-license-files = ["LICENSE", "LICENSE.mecab"]
```

For more information, please check the following URL
https://packaging.python.org/en/latest/guides/licensing-examples-and-user-scenarios/

Reply:
`tool.setuptools.license-files` will be moved to `project.license-files` instead of being removed.